community/modules/scripts/ramble-setup/main.tf

```diff
@@ -45,16 +45,31 @@ locals {
   )
 
   deps_content = templatefile(
-    "${path.module}/templates/install_ramble_deps.yml.tpl",
+    "${path.module}/templates/install_ramble_deps.yml.tftpl",
     {
-      ramble_ref = var.ramble_ref
+      virtualenv_path = var.ramble_virtualenv_path
     }
   )
 
   install_ramble_deps_runner = {
     "type"        = "ansible-local"
     "content"     = local.deps_content
     "destination" = "install_ramble_deps.yml"
+    "args"        = "-e virtualenv_path=${var.ramble_virtualenv_path}"
+  }
+
+  python_reqs_content = templatefile(
+    "${path.module}/templates/install_ramble_python_deps.yml.tftpl",
+    {
+      install_dir     = var.install_dir
+      virtualenv_path = var.ramble_virtualenv_path
+    }
+  )
+
+  python_reqs_runner = {
+    "type"        = "ansible-local"
+    "content"     = local.python_reqs_content
+    "destination" = "install_ramble_reqs.yml"
     "args"        = "-e ramble_virtualenv_path=${var.ramble_virtualenv_path}"
```

Comment:
I believe this arg has no effect as this value is not used in install_ramble_python_deps.